nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -681,11 +654,28 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(IAccessible2* pacc,
 		paccTableCell=nullptr;
 	}
 
-	if(paccTableCell) {
-		LOG_DEBUG(L"Setting node's requiresParentUpdate back to false as this is a table cell");
-		parentNode->requiresParentUpdate=false;
-		LOG_DEBUG(L"Setting node's alwaysRerenderChildren back to false as this is a table cell");
-		parentNode->alwaysRerenderChildren=false;
+	if(paccTable2) {
+		// We are rendering a node that is part of a table (row group, row or cell).
+		// Set some properties to ensure that this and other nodes in the table aare correctly re-rendered if the table changes,
+		// so that the table's row and column cordinates remain accurate.
+		// setting denyReuseIfPreviousSiblingsChange ensures that if any part of the table is added or removed previous to this node,
+		// his node will not be reused (as its row / column coordinates would now be out of date).
+		LOG_DEBUG(L"Setting node's denyReuseIfPreviousSiblingsChanged to true");
+		parentNode->denyReuseIfPreviousSiblingsChanged=true;
+		if(!paccTableCell) { // just rows and row groups
+			// setting requiresParentUpdate ensures that if this node is specifically invalidated,
+			// its parent will also be invalidated.
+			// For example, if this is a table row, its rerendering may change the number of cells inside. 

```

Comment:
Re-rendering a table row wouldn't change subsequent table row coordinates. However, this does apply to row groups, so you can just change the comment here (and on the next line) to reflect that.